packages/next/build/polyfills/fetch/whatwg-fetch.js

```diff
@@ -0,0 +1,10 @@
+/* globals self */
+const fetch = {
+  Headers: self.Headers.bind(self),
```

Comment:
Do these need to be bound? We're not decorating any properties on them.
```suggestion
  Headers: self.Headers,
```